apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjFissDiagnosisCode.java

```diff
@@ -49,12 +50,13 @@
   @Column(name = "`lastUpdated`")
   private Instant lastUpdated;
 
+  /** PK class for the FissDiagnosisCodes table */
   @Data
   @NoArgsConstructor
   @AllArgsConstructor
-  /* PK class for the FissDiagnosisCodes table */
```

Comment:
I had never thought much about it, but after some research putting the documentation before the annotations is the recommended way to document. TIL! 🎉